docs/source/tutorial/multi_gpu_vanilla.rst

```diff
@@ -0,0 +1,323 @@
+Multi-GPU Training in Pure PyTorch
+==================================
+
+For many large scale, real-world datasets, it may be necessary to scale-up training across multiple GPUs.
+This tutorial goes over how to do this on a small scale dataset and then shows how the same technique can be applied for large-scale recommendation systems.
+In particular, this tutorial introduces how to utilize :pyg:`PyG` with pure :pytorch:`PyTorch` for multi-GPU training via :class:`torch.nn.parallel.DistributedDataParallel`, without the need for any other third-party libraries (such as :lightning:`PyTorch Lightning`).
+
+Starting off small
+------------------
+
+To start, we can take a look at the `distributed sampling <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/multi_gpu/distributed_sampling.py>`__ example from :pyg:`PyG`.
+This example shows how to use train a :class:`~torch_geometric.nn.models.GraphSAGE` GNN model on the :class:`~torch_geometric.datasets.Reddit` dataset.
+This example uses the :class:`~torch_geometric.loader.NeighborLoader` with :class:`torch.nn.parallel.DistributedDataParallel` to scale-up training across all available GPU's on your machine.
+
+Defining our GNN
+~~~~~~~~~~~~~~~~
+
+.. code-block:: python
```

Comment:
Can we use the `GraphSAGE` model here from `torch_geometric.nn.models` directly? I think the focus of this tutorial shouldn't be on how to build your model.